tests/integration/cli_test.py

```diff
@@ -258,18 +258,18 @@ def test_run_service_with_entrypoint_overridden(self, _):
             [u'/bin/echo', u'helloworld'],
         )
 
-    @patch('dockerpty.start')
+    @mock.patch('dockerpty.start')
     def test_run_service_with_user_overridden(self, _):
         self.command.base_dir = 'tests/fixtures/user-composefile'
         name = 'service'
         user = 'sshd'
-        args = ['run', '--user={}'.format(user), name]
+        args = ['run', '--user={user}'.format(user=user), name]
```

Comment:
i think this change isn't needed if Py2.6 is not supported. but it isn't wrong neither.